src/components/Home.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable max-len */
```

Comment:
- [ ] Let's don't disable linter rules. These rules are here to help us write the best code.